.github/workflows/gradle.yml

```diff
@@ -242,20 +246,26 @@ jobs:
           attempt_limit: 3
           attempt_delay: 5000 # in ms
 
-      # Verify that the linter is happy
```

Comment:
to be safe, instead of fully removing this, I think we should move this step above the format step, to ensure that nothing in the previous build steps accidentally caused code to be added. The only thing which should cause a git diff is formatting/resources etc.. (it should be more of an allow-list model of what is permitted to muddy git diffs)